src/wrapper/wrapper.c

```diff
@@ -238,7 +180,8 @@ FILE *fopen (const char *path, const char *mode)
     }
 
     if (!(ctx && ctx->h) || (ctx && !ctx->reenter) || !path) {
-        IPRINTF (ctx, "DYAD_SYNC: fopen sync not applicable for \"%s\".\n",
+        IPRINTF (ctx,
```

Comment:
DYAD_LOG_INFO

Reply:
We should explore unifying the logging mechanisms in DYAD. I'll create an issue for that so we can track it as future work.

Reply:
I created #44 to track this.